### ISSUE TITLE
ReferenceError: Can't find variable: Buffer

### ISSUE THREAD

**Issue by nvzsolutions** [Avansai/properties-file#12]
Hi!

I was exited to see a maintained package for dealing with property files.
When i install it into a brand new vite project and try to call getProperties i get an error in my browser that Buffer is not found? Missing Dependency maybe?

```
import { getProperties } from 'properties-file'

console.log(getProperties("hello = hello1\nworld = world1\nworld = world2"))
```

<img width="928" alt="Screenshot 2023-04-28 at 22 14 23" src="https://user-images.githubusercontent.com/6881597/235244870-bb95b22f-f776-415b-8381-33ffbd1788f8.png">

**Comment by nbouvrette** [comment]
Thanks for reporting this. Can you share more details on how you are triggering this error? For example is this on Node.js and which version?

I should be able to fix this pretty quickly.

**Comment by nbouvrette** [comment]
@nvzsolutions this should be fixed in version 3.1.1 - thanks for reporting this issue.
